mathesar_ui/src/components/sheet/Sheet.svelte

```diff
@@ -104,12 +104,12 @@
     display: flex;
     flex-direction: column;
     isolation: isolate;
-    --z-index__sheet__active-cell: 1;
     --z-index__sheet__column-resizer: 2;
     --z-index__sheet__top-left-cell: 3;
+    --z-index__sheet_group-header: 3;
+    --z-index__sheet_new-record-message: 3;
+    --z-index__sheet__active-cell: 4;
```

Comment:
I like that you've made changes to these property names to make them more consistent with the other properties, but I see only a single underscore after "sheet" in two of the properties here. It would be nice to follow the naming scheme exactly, using double underscores around "sheet".

This is super minor though, and we can make this change later.